Project.toml

```
@@ -0,0 +1,14 @@
+name = "OrderedCollections"
+uuid = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
+version = "2.0.0-DEV"
+
+[compat]
+julia = "≥ 0.7.0"
```

```suggestion
julia = "0.7, 1"
```
Will likely break with Julia 2.0 so might as well prevent problems now.

---

Please go ahead and change what you want. I've no idea about Julia 2.0 !

---

Just click the button. I don't have permission to modify your repo.